website/templates/include/comment_pane_template.mako

```diff
@@ -15,9 +15,12 @@
                 <i class="fa fa-times"></i>
             </button>
             <h4>
-                <span data-bind="if: page() == 'node' ">${node['title']} Discussion</span>
                 %if file_name:
                     <span data-bind="if: page() == 'files'">Files | ${file_name} Discussion</span>
+                %elif wiki_name:
+                    <span data-bind="if: page() == 'wiki'">Wiki | ${wiki_name} Discussion</span>
```

Comment:
When I remove the mako conditionals, mako throws an error because `file_page` is undefined (unless you're viewing the file detail page)